sfts/s3_to_sfts.py

```diff
@@ -161,19 +225,23 @@ def is_processed():
 try:
     xfer_jar = f"{xfer_path}/xfer.jar"
     jna_jar = f"{xfer_path}/jna.jar"
-    print(("trying to call subprocess:\nxfer.jar: "
+    logger.info(("trying to call subprocess:\nxfer.jar: "
            f"{xfer_jar}\njna.jar : {jna_jar}"))
-    subprocess.check_call(
+    output = subprocess.check_output(
```

Comment:
@lclarko `output` is a byte literal (see: https://docs.python.org/3/library/stdtypes.html#binary-sequence-types-bytes-bytearray-memoryview).

This results in logs that will be difficult to read at a glance:
```
INFO:__main__:2021-05-13 11:03:05,994:b'User GDXSFTSDEV signed on\n--> cd GCPE/Shared/gdxtest\n--> put ./tmp/test_pmrp_20200228_20200327_20200330T183405_part000.csv\nUpload complete. File ID 767097304\n125126 bytes sent in 1.94 seconds at 62.99 kB/s\n--> quit\n'
```
You'll see there that the output is prepended with `b`, confirming a byte literal.

There are flags for `subprocess.check_output` that should resolve this issue.

From https://docs.python.org/3/library/subprocess.html#frequently-used-arguments
> If text mode is not used, stdin, stdout and stderr will be opened as binary streams. No encoding or line ending conversion is performed.

Please try setting the `text` flag true. You might also need to set the `encoding` flag; I'm not sure.

Reply:
decoding on log output instead.